### PR TITLE
Add appclient test cases to JPA FVT bucket.

### PR DESCRIPTION
**Pull request by jgrassel**
Add JPA application client tests to com.ibm.ws.jpa_22_fat, tests appclient @PersistenceUnit injection into classes, fields, and methods.

Signed-off-by: Joe Grassel <jgrassel@us.ibm.com>

### CODE REVIEW

**Comment by jgrassel** [comment]
#build

**Comment by jgrassel** [comment]
Connecting with issue #458 since no product changes were needed, and this fat tests the function identified by the issue.

File: dev/com.ibm.ws.jpa_22_fat/fat/src/com/ibm/ws/jpa/JPAAppClientTest.java [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+public class JPAAppClientTest {
+    private static final String APPCLIENT_ROOT = "test-applications/appclient";
+    private final static String CLIENT_NAME = "JPAAppClient";
+    private final static String PKG_ROOT = "jpaappcli";
+
+    protected static LibertyClient client = LibertyClientFactory.getLibertyClient(CLIENT_NAME);
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+
```
Comment (k8vance88) [inline dev/com.ibm.ws.jpa_22_fat/fat/src/com/ibm/ws/jpa/JPAAppClientTest.java]: Do we need these methods even though they are empty?

Comment (jgrassel) [reply dev/com.ibm.ws.jpa_22_fat/fat/src/com/ibm/ws/jpa/JPAAppClientTest.java]: We probably don't, I just left them in place should I change my mind later and add content.  I don't think it hurts anything.

**Pull request merged**